express/styles/styles.css

```diff
@@ -544,6 +552,19 @@ main .section:last-of-type>.default-content-wrapper {
   main p {
     font-size: var(--body-font-size-l);
   }
+
+  main .section:first-of-type {
+    margin-bottom: 0;
```

Comment:
Feels risky here too. This affects all of the first sections across the entire site. Maybe I'm missing the picture. What's the ask behind this?